### DISCUSSION TITLE
Apple TV 使用体验及相关问题

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#113] [Announcements]
https://www.xheldon.com/life/apple-tv-using-and-problem.html 

 「就是把钱扔水里听响也绝不给国内的某些厂商」——媳妇儿说

**Comment by lunazzm** [comment]
感谢，非常受益。想再确认一下，apple tv可以登录国区账号，在电视上欣赏自己的照片视频，是这样吧？

**Comment by Xheldon** [comment]
> 感谢，非常受益。想再确认一下，apple tv可以登录国区账号，在电视上欣赏自己的照片视频，是这样吧？

是的，没错

**Comment by wentao1031** [comment]
咨询播主下：
我是用梅林科学上网的，在apple tv 盒子上使用apple相关服务时（apple TV 、music等）流量会走代理。怎么设置让apple 相关服务不走代理？

**Reply by Xheldon** [reply]
这个需要视你家里的拓扑网络结构而定的，不过一般来说，在梅林的翻墙插件里面设置特定设备或者特定域名的不走代理（一般机场会提供分流设置）就行了。

**Comment by Xheldon** [comment]
更新，好消息！
1、tvOS 17 现在支持安装 VPN 应用了，不用再折腾软路由，上手成本更低，更值得入手了。
2、infuse 现在支持直接连接阿里云盘，不用再折腾软路由的 WebDav，上手成本更低，新加到云盘的视频也可以直接刷新出来不用再等一段时间了，体验更好，更值得入手了。

**Reply by horizon365** [reply]
请问，能在tv上看自己 icoud 文件或者相册里存的电影吗？

**Reply by Xheldon** [reply]
可以，电影能在 iPhone 的相册播放就能在 Apple TV 的相册播放
